src/custom-surf/pages/ServiceTickets.tsx

```diff
@@ -295,7 +317,19 @@ class ServiceTickets extends React.PureComponent<IProps, IState> {
                                         data-label={intl.formatMessage({ id: "tickets.table.start_date" })}
                                         className="start_date"
                                     >
-                                        {renderStringAsDateTime(ticket.start_date)}
+                                        {renderStringAsDateTime(ticket.start_date, true)}
+                                    </td>
+                                    <td
+                                        data-label={intl.formatMessage({ id: "tickets.table.updated_on" })}
+                                        className="create_date"
```

Comment:
`className="update_date"` ?

Reply:
No it's a class: it's a date and we could style them all the same.
But they are also used in a loop to style the headers themselves. So changed it.

Also ensured that all columns together: 100%